x-pack/plugins/apm/public/components/app/backend_detail_overview/backend_detail_dependencies_table.tsx

```diff
@@ -83,6 +83,7 @@ export function BackendDetailDependenciesTable() {
               rangeTo,
               latencyAggregationType: undefined,
               transactionType: undefined,
+              serviceGroup: '',
```

Comment:
Why are we setting this to an empty string. I'd expect this to either have a value or be `undefined`. This feels like a hack but maybe I don't know enough

Reply:
@dgieselaar gave early feedback to make `serviceGroup` a required query param for the relevant routes with the suggestion to explicitly set an empty serviceGroup on those routes.

Reply:
I think the idea was to have it be set or null, but I can't remember the details to be honest 😄